### PR TITLE
First pass at QA checklist

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
Thoughts: should we default to the localhost server or the staging server for QA?

File: docs/QA_Checklist.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# QA Checklist
+
+## Sandbox account credentials
+
+If you are using the PayPal or Stripe sandbox accounts in [sample.env](../sample.env) then you can use the following credentials/credit cards for QA
+
+### Paypal accounts
+
+send-donation@test.com / testtest
```
Comment (ScottDowne) [inline docs/QA_Checklist.md]: Is it clear that this is name/password?

Why not just do 
name: send-donation@test.com
password: testtest

File: docs/QA_Checklist.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- City - Any city ie **Toronto**
+- Zip/Postal code - Any zip/postal code ie **M5V 1R9**
+- State/Province - Any state/province ie **Ontario**
+- Email - Any email ie **maryroe@example.org**
+
+Stripe also provides many other [test credit cards](https://stripe.com/docs/testing#cards) for other brands and Stripe error messages.
+
+## Flows
+
+### PayPal flow
```
Comment (ScottDowne) [inline docs/QA_Checklist.md]: Should we have a slightly modified flow for testing recurring donations for stripe and PayPal?

File: docs/QA_Checklist.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# QA Checklist
```
Comment (ScottDowne) [inline docs/QA_Checklist.md]: Is this a generic QA checklist that is just starting, or is this a complete QA for just payment processora/signup API?

Comment (jbuck) [inline docs/QA_Checklist.md]: It's a QA checklist for everything that should work on the site

Comment (ScottDowne) [inline docs/QA_Checklist.md]: Works for me.

**Comment by jbuck** [comment]
Fix #256

**Pull request merged**